packages/neutrino-preset-react/index.js

```diff
@@ -39,8 +40,19 @@ module.exports = (neutrino) => {
       'react/lib/ReactContext': 'window'
     })
     .when(process.env.NODE_ENV === 'development', config => config
+      .module
```

Comment:
We could combine this with the previous babel loader merge at the beginning of this preset:

```diff
  neutrino.use(loaderMerge('compile', 'babel'), {
    presets: [require.resolve('babel-preset-react')],
    plugins: [require.resolve('babel-plugin-transform-object-rest-spread')],
    env: {
      development: {
        plugins: [
          require.resolve('react-hot-loader/babel'),
+         require.resolve('babel-plugin-transform-es2015-classes')
        ]
      }
    }
  });
```